src/relax/op/tensor/create.cc

```diff
@@ -144,7 +146,8 @@ TVM_REGISTER_OP("relax.ones")
     .set_attrs_type<InitAttrs>()
     .set_num_inputs(1)
     .add_argument("shape", "ShapeExpr", "The shape of the created tensor.")
-    .set_attr<FInferStructInfo>("FInferStructInfo", InferStructInfoOnesZeros);
+    .set_attr<FInferStructInfo>("FInferStructInfo", InferStructInfoOnesZeros)
+    .set_attr<FMixedPrecision>("FMixedPrecision", InferMixedPrecisionFollow);
```

Comment:
Do we need to register `FMixedPrecision` for all ops? I guess only interested ops are needed, i.e. gemm and conv. And left other ops as default.

Reply:
There are 3 policies. 

ALWAYS: Always use fp16
NEVER:   Always use fp32
FOLLOW: follow the requirement of consumer

Now the default is NEVER.

Reply:
How about changing the default to FOLLOW? Given Torch use FOLLOW as default: https://pytorch.org/docs/stable/amp.html#cuda-op-specific-behavior